R/pkg/R/mllib.R

```diff
@@ -197,11 +197,10 @@ print.summary.GeneralizedLinearRegressionModel <- function(x, ...) {
   invisible(x)
   }
 
-#' Make predictions from a generalized linear model
-#'
 #' Makes predictions from a generalized linear model produced by glm() or spark.glm(),
 #' similarly to R's predict().
 #'
+#' @title predict
```

Comment:
I personally prefer using the first sentence as the convention to define the title. One thing though is I'm not sure how the titles get collated when we put multiple functions in the same doc page. @felixcheung do you know if using `@title` would make it easier to have a canonical title in that case ?